database/index.js

```diff
@@ -57,35 +57,53 @@ exports.migrate = {
     return 1;
   },
 
-  teardown: async (name, env = process.env.NODE_ENV) => {
+  teardown: async ({env = process.env.NODE_ENV, client} = {}) => { 
     const config = getConfig(env);
-
-    const client = knex(config.knex);
-
-    const tables = await client('pg_catalog.pg_tables').select('tablename').where({ schemaname: 'public' });
+    const teardownClient = client ? client : knex(config.knex);
+    
+    const tables = await teardownClient('pg_catalog.pg_tables').select('tablename').where({ schemaname: 'public' });
 
     const queries = tables
       .map((t) => t.tablename)
       .filter((t) => t !== 'spatial_ref_sys')
-      .map(tap((t) => console.log(`Dropping table ${t}`)))
-      .map((tablename) => client.raw(`DROP TABLE IF EXISTS "${tablename}" CASCADE`))
+      .map(tap((t) => { 
+        if(config.env !== 'testing') console.log(`Dropping table ${t}`)
```

Comment:
The permitted environment values are enumerated as constants in the config module, so lets import them and use them here instead of hardcoding the value.